com.microsoft.mrtk.uxcomponents/SeeItSayItLabel/SeeItSayItLabel.prefab

```diff
@@ -0,0 +1,316 @@
+%YAML 1.1
```

Comment:
I don't see a Canvas SeeItSayIt component here.... is it somewhere else?

Reply:
Added canvas version:
![2022-08-05 16_40_17](https://user-images.githubusercontent.com/13754172/183027491-7d62deac-c2d6-40d9-aa9c-618aadbe40db.png)